src/rred_reports/redcap/main.py

```diff
@@ -58,6 +62,8 @@ def read_single_redcap_year(self, redcap_fields: ExtractInput) -> pd.DataFrame:
         raw_data = pd.read_csv(redcap_fields.coded_data_path, low_memory=False)
         labelled_data = pd.read_csv(redcap_fields.labelled_data_path, low_memory=False)
         processed_wide = self.preprocess_wide_data(raw_data, labelled_data)
+        if self._school_aliases:
+            processed_wide["school_id"] = processed_wide["school_id"].replace(self._school_aliases)
```

Comment:
I think just need to choose one place for it rather than here and down in `add_school_name_column`